src/ui/services/Background.ts

```diff
@@ -229,6 +231,10 @@ class Background {
     }
   }
 
+  async nftInfo(): Promise<Asset[]> {
```

Comment:
Looks like it actually returns plain objects instead of `Asset` instances. Or do I miss something?